pkg/procmatch/contains_matcher.go

```diff
@@ -0,0 +1,27 @@
+package procmatch
+
+import (
+	"strings"
+)
+
+type containsMatcher struct {
```

Comment:
We found that there was a high rate of false positives with the `containsMatcher`, so lets remove this and only leave the graph-based matcher